docs/mxnet_backend/save_mxnet_model.md

```diff
@@ -3,32 +3,41 @@
 ## Table of Contents
 
 1. [Objective](#objective)
-2. [Train and Save a Convolutional Neural Network (CNN) Model for MNIST Dataset](#train-and-save-a-convolutional-neural-network-cnn-model-for-mnist-dataset)
-3. [Import the model in MXNet for Inference](#import-the-model-in-mxnet-for-inference)
-4. [References](#references)
+2. [Export MXNet model using save_mxnet_model() API](#export-mxnet-model-using-save_mxnet_model-api)
+3. [Checkpoint MXNet model using MXNetModelCheckpoint callback](#checkpoint-mxnet-model-using-mxnetmodelcheckpoint-callback)
+4. [Train and Save a Convolutional Neural Network (CNN) Model for MNIST Dataset](#train-and-save-a-convolutional-neural-network-cnn-model-for-mnist-dataset)
+5. [Import the model in MXNet for Inference](#import-the-model-in-mxnet-for-inference)
+6. [References](#references)
 
 ## Objective
 
-In this tutorial, we show how to train a model in Keras-MXNet, export the trained model as Apache MXNet model using `keras.models.save_mxnet_model()` API, and use MXNet natively for inference.
+In this tutorial, we show how to train a model in Keras-MXNet, export the trained model as Apache MXNet model, and use MXNet natively for inference.
 
-The Keras interface is known for its easy to use APIs, enabling fast prototyping in deep learning research. 
-MXNet is known for its high performance, production ready engine. With Keras-MXNet, you get an out-of-the-box API to 
-export most trained Keras models in MXNet model format. 
-You can now use Keras-MXNet for training the model and MXNet for inference in production. You can use `keras.models.save_mxnet_model()` API to save 
-the models trained on the CPU, a single GPU or multiple GPUs.
+You can use one of the below APIs to export the native MXNet model from Keras-MXNet:
+1. Using `keras.models.save_mxnet_model()` API.
+2. Using `keras.callbacks.MXNetModelCheckpoint` callback.
+
+The Keras interface is known for its easy to use APIs, enabling fast prototyping in deep learning research.
+MXNet is known for its high performance, production ready engine. With Keras-MXNet, you get an out-of-the-box API to
+export trained Keras models in MXNet model format.
+You can now use Keras-MXNet for training the model and MXNet for inference in production. You can use `keras.models.save_mxnet_model()` API to save
+the models trained on the CPU, a single GPU or multiple GPUs or, you can use `keras.callbacks.MXNetModelCheckpoint` callback to checkpoint the model.
 
 You can use any language bindings supported by MXNet (Scala/Python/Julia/C++/R/Perl) for performing inference with these models!
 
 `Warning` Not all Keras operators and functionalities are supported with MXNet backend. For more information, view the the list
  of known issues and unsupported functionalities [here](https://github.com/awslabs/keras-apache-mxnet/issues/18).
 
+
+## Export MXNet model using save_mxnet_model() API
```

Comment:
shall we add links to both the scala & MMS blog posts to show how the exported model can be further used with another Deep learning utility?

Reply:
see updates in #151